io.github.georgewoodall82.jsymphonic.yml

```diff
@@ -0,0 +1,33 @@
+id: io.github.georgewoodall82.jsymphonic
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=host
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk21
+build-options:
+  build-args:
+    - --share=network
```

Comment:
this won't work. Build MUST be offline. It can't download anything that the manifest doesn't know.

Reply:
fixed now in  latest commit